wof/__init__.py

```diff
@@ -7,3 +7,10 @@
 
 from wof.core_1_0 import WOF
 from wof.core_1_1 import WOF_1_1
+
+__all__ = [
```

Comment:
`__all__` is what you get when using `import *`. Even though we should not use `import *` it is nice to define it when bring submodules into a namespace like it is done here.